examples/lock-app/lock-common/BUILD.gn

```diff
@@ -17,11 +17,14 @@ import("//build_overrides/chip.gni")
 import("${chip_root}/src/app/chip_data_model.gni")
 
 chip_data_model("lock-common") {
-  cluster_sources = [
-    "bindings",
-    "network-commissioning",
-    "on-off-server",
-  ]
+  zap_file = "lock-app.zap"
 
   zap_pregenerated_dir = "gen"
+
+  # These clusters are configured in the ZAP file but don't compile.
+  remove_cluster_sources = [
+    "groups-server",
+    "identify",
+    "scenes",
```

Comment:
Groups should not be configured for lock.  Or scenes, probably.  "identify" I don't know..